pulpcore/cli/rpm/context.py

```diff
@@ -33,6 +35,21 @@ def refresh(self, href: str) -> Any:
         )
 
 
+class PulpRpmCompsXmlContext(PulpEntityContext):
+    UPLOAD_COMPS_ID = "upload_comps"
```

Comment:
```suggestion
    UPLOAD_COMPS_ID: ClassVar[str] = "upload_comps"
```
Is there any reason not to call this `UPLOAD_ID`?

Reply:
> Is there any reason not to call this `UPLOAD_ID`?

Mmm, I believe the original intent was to differentiate from the uploading of model-entities like Package and Advisory. Prob not useful, will change.

Reply:
Actually, i kind of think it should be `rpm_comps_upload` on the server side...

Reply:
Hrm, that was my initial id, but then I changed it to conform to copy - I should have stuck w/ my first thought, I suppose. Well, the PR is waiting for final stakeholder approval, so we can fixit!